prombench/manifests/prombench/benchmark/2a_prometheus-secret.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-secret
+  namespace: prombench-{{ .PR_NUMBER }}  # Replace with your actual namespace
+type: Opaque
+stringData:
+  object-config.yml: |
+    type: S3
```

Comment:
This is going to be configured differently for each installation - `s3` is good for local installs using Minio, but when run at Google Cloud it needs to say `gcs`.
I suggest you take this Secret out of the PR and instead put instructions in the docs, e.g. [here](https://github.com/prometheus/test-infra/blob/master/prombench/docs/gke.md), for the person who installs Prombench to set it up.